files/settings-default.cfg

```diff
@@ -642,8 +642,8 @@ enable = true
 # Scale of NavMesh coordinates to world coordinates (value > 0.0). Recastnavigation builds voxels for world geometry.
 # Basically voxel size is 1 / "cell size". To reduce amount of voxels we apply scale factor, to make voxel size
 # "recast scale factor" / "cell size". Default value calculates by this equation:
-# sStepSizeUp * "recast scale factor" / "cell size" = 3 (max climb height should be equal to 3 voxels)
-recast scale factor = 0.017647058823529415
+# sStepSizeUp * "recast scale factor" / "cell size" = 4 (max climb height should be equal to 4 voxels)
+recast scale factor = 0.023529411764705882
```

Comment:
I suppose the ``"recast scale factor" = (numVoxels * cellSize) / sStepSizeUp``.

Why just do not store a count of voxels as an integer setting instead of irrational number?

Reply:
It's possible to have non integer number of voxels per `sStepSizeUp`. So far my assumption is that it's better to have this value aligned to `sStepSizeUp` because it is better with slope edge cases. I haven't done much testing for this. Basically this affects slopes and roofed locations.

In theory this is a binary classification in therms of machine learning. By changing the value we have following situations:
* Passable by physics and passable by navmesh (true positive)
* Passable by physics and impassable by navmesh (false negative)
* Impassable by physics and passable by navmesh (false positive)
* Impassable by physics and impassable by navmesh (true negative)

To get optimal value we need to run optimization over "all" possible slopes and roofed locations maximize [F-score](https://en.wikipedia.org/wiki/F1_score). Also we need to consider that by increasing `recast scale factor` real size of each tile decreases. Having limited amout of tiles this decreases area covered by navmesh. That's where [64bit navmesh](https://github.com/OpenMW/openmw/pull/2229) may give some benefit. It'd be much of fun to setup such an environment to make this optimization possible in openmw.

Reply:
> It's possible to have non integer number of voxels per sStepSizeUp.

Even in this case it is simplier so store it directly as a setting, rather than storing an arbitrary irrational multiplier to calculate the same value.

Reply:
This value is not calculated anywhere except probably recastnavigation internals, search for mRecastScaleFactor to see.

Reply:
Voxel size is a constant = 1.